src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/CosmosFhirDataStore.cs

```diff
@@ -278,7 +279,15 @@ internal async Task<FeedResponse<T>> ExecuteDocumentQueryAsync<T>(QueryDefinitio
 
             var documentQuery = _cosmosQueryFactory.Create<T>(_containerScope.Value, context);
 
-            return await documentQuery.ExecuteNextAsync(cancellationToken);
+            try
+            {
+                return await documentQuery.ExecuteNextAsync(cancellationToken);
+            }
+            catch (CosmosException e) when (e.StatusCode == HttpStatusCode.BadRequest && continuationToken != null && e.ResponseBody.StartsWith("Malformed continuation token", StringComparison.OrdinalIgnoreCase))
```

Comment:
Did this pass through https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/CosmosResponseProcessor.cs#L45? Should this be in the handler?

Reply:
Makes sense. I was thinking that we should only return a 400 if this was a CT that came from a client, not one that we used internally. But that is probably unlikely.

Reply:
@brandonpollett, it does not pass though the handler:
![image](https://user-images.githubusercontent.com/339432/97034343-c43fdd80-1532-11eb-94bc-3e10c7172240.png)
I don't know if this is a bug in the SDK or our code.

Reply:
Thanks, I'll make a bug to look into this

Reply:
#1390